src/main/io/displayport_crsf.c

```diff
@@ -192,7 +192,7 @@ bool crsfDisplayPortIsReady(void)
 {
     const timeMs_t currentTimeMs = millis();
     const bool delayExpired = (currentTimeMs > delayTransportUntilMs);
-    const bool cmsReady = (cmsInMenu && (pCurrentDisplay = &crsfDisplayPort));
+    const bool cmsReady = (cmsInMenu && ((pCurrentDisplay == &crsfDisplayPort) != NULL));
```

Comment:
Just `cmsInMenu && pCurrentDisplay == &crsfDisplayPort`
The `(pCurrentDisplay = &crsfDisplayPort) != NULL` is way to explicitly tell that assignment into pCurrentDisplay is intentional